stock_picking_late_activity/models/stock_picking.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    create_late_picking_activity = fields.Boolean(
+        help="If checked, the pickings belonging to this 'operation "
+             "type' will be checked periodically to request to update "
+             "their planned date.",
+    )
+    late_picking_activity_user_id = fields.Many2one(
+        comodel_name="res.users",
+        string="Late picking activity responsible",
+    )
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _cron_late_picking_activity(self):
+        """ Create an activity for late pickings. """
+        late_picking = self.search([
+            ("state", 'not in', ["done", "cancel"]),
+            ("picking_type_id.create_late_picking_activity", '=', True),
+            ("scheduled_date", '<', datetime.now()),
+        ])
+        if not late_picking:
+            return False
+        activity_type = self.env.ref(
+            "stock_picking_late_activity"
+            ".mail_activity_type_update_scheduled_date")
+        vals_list = []
+        for picking in late_picking:
+            if not bool(picking.activity_ids.filtered(
+                    lambda r: r.activity_type_id == activity_type)):
+                vals_list.append(picking._prepare_late_picking_activity_vals())
+        self.env["mail.activity"].create(vals_list)
+
+    def _prepare_late_picking_activity_vals(self):
+        """ Hook method to prepare the values to create a late picking
+        activity."""
```

Comment:
Follow proper PEP257 for docstring
```suggestion
        activity.
        """
```